### DISCUSSION TITLE
Force reloading translations

### DISCUSSION THREAD

**Discussion by Raphiiko** [sveltekit-i18n/lib#131] [Q&A] [answered]
Translations that have been previously loaded are not loaded again. This makes sense. However, I have a use case where a translation file might be updated, and I want to force them to be reloaded without doing a full page reload.

Is there a way to maybe unload translations that have already been loaded previously so that the loader is called again?

**Comment by jarda-svoboda** [comment, marked as answer]
Hi @Raphiiko!
Have you tried to [set cache](https://github.com/sveltekit-i18n/base/tree/master/docs#cache-number) to `0`?

**Reply by Raphiiko** [reply]
Hi! Thank you for the quick answer, this indeed works for me!

> When you are running your app on Node.js server, translations are loaded only once during the SSR. This property allows you to setup a refresh period in milliseconds when your translations are refetched on the server.

I had initially dismissed the option because of this description. I assumed it would not apply, considering my SvelteKit app has all SSR disabled, uses `@sveltejs/adapter-static`, and is therefore only served statically without any Node.JS server. I should probably have checked anyways. 

Thanks again!

**Reply by jarda-svoboda** [reply]
Thanks, that’s a good point. I’ll update the description.

[a comment by an automated account is not shown]
